profiler/internal/stackparse/test-fixtures/empty_input.golden.json

```diff
@@ -0,0 +1,4 @@
+{
+  "Errors": null,
+  "Goroutines": null
+}
```

Comment:
Nit: +newlines in a few places

Reply:
Hm, considering that these are auto-generated files that nobody should edit, I'm okay with the lack of trailing newlines. But if you want I'm happy to submit a follow-up PR to fix this?

Reply:
Nvm, I didn't have "approval" to merge anymore anyway, so here we go: e335ebe84769bab20835053a3280912facd8076b